homeassistant/components/pushover/notify.py

```diff
@@ -67,7 +67,7 @@ def send_message(self, message="", **kwargs):
                         # Replace the attachment identifier with file object.
                         data[ATTR_ATTACHMENT] = response.content
                     else:
-                        _LOGGER.error("Image not found")
+                        _LOGGER.error(f"Image '{data[ATTR_ATTACHMENT]}' not found: {response.status_code}")
```

Comment:
Please use the following format for error messages:
```python
_LOGGER.error("Image %s not found, response status code: %s", data[ATTR_ATTACHMENT], response.status_code)
```
See the last example in the [style guidelines](https://developers.home-assistant.io/docs/en/next/development_guidelines.html) for more information.